base/libgit2/libgit2.jl

```diff
@@ -484,7 +484,17 @@ function cat(repo::GitRepo, spec)
     end
 end
 
-""" git rev-list --count <commit1> <commit2> """
+"""
+    LibGit2.revcount(repo::GitRepo, fst::AbstractString, snd::AbstractString)
```

Comment:
The arg names are mentioned in the docstring, both in prose and in the command line Git equivalent at the bottom. Would `left` and `right` be accurate names?